downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md

```diff
@@ -74,8 +90,18 @@ To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://g
 
 ### New features
 
+#### Persist Tool
+
+- Added support for the `In-Memory` data store. Set as the default data store.
+- Added support for the `Google Sheets` data store. The support for Google Sheets data store is currently an experimental feature, and its behavior may be subject to change in future releases.
```

Comment:
```suggestion
- Added support for the `Google Sheets` data store. This is currently an experimental feature and its behavior may be subject to change in future releases.
```
Why is this repeated above also?

Reply:
We added this support in both module and tooling side